test-utils/loadtester/src/stats.rs

```diff
@@ -0,0 +1,91 @@
+use crate::remote_node::{get_result, RemoteNode};
```

Comment:
I believe `test-utils/loadtester` is removed in master. What is the reason for restoring it?

Reply:
no reason, it looks like it happened by accident